app/renderer/components/Inspector/Commands.js

```diff
@@ -24,6 +24,19 @@ const Commands = (props) => {
     // Make a copy of the arguments to avoid state mutation
     let copiedArgs = _.cloneDeep(args);
 
+    let parsingSuccess = true;
+    const parseJsonArgument = (index) => {
+      try {
+        copiedArgs[index] = JSON.parse(args[index]);
```

Comment:
Closures might be dangerous if not used carefully. maybe we could simply move it to a separate function and return object/tuple instead?

Reply:
Moved to a separate function